### ISSUE TITLE
Stalled post to pgsql-jdbc

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11415]
|<img alt="pgsql-jdbc-owner's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| pgsql-jdbc-owner reported|
|-|-|
|Trac ID|trac#568|
|Type|enhancement|
|Created|2004-03-31 04:26:32Z|

Attachments:

* [[issue564] warning](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/8a1/8a1beaee63fed318ca54d4f7d18cee4081c68a74/c8f74e90ef9e65083a82f8d7fedbc55b81277e55) (2003 bytes) - added by pgsql-jdbc-owner on 2004-03-31 04:26:32Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__568 568
type__enhancement enhancement
reporter__pgsql_jdbc_owner pgsql-jdbc-owner
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1080707192000000 1080707192000000
changetime__1477905579863317 1477905579863317
version__ 
owner__ 
cc__pgsql-jdbc-owner
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="pgsql-jdbc-owner's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|pgsql-jdbc-owner commented|
|-|-|

```
#!html
<pre>
Your message to pgsql-jdbc has been delayed, and requires the approval 
of the moderators, for the following reason(s):

The author ("Chris Armstrong [Twisted issue tracker]" &lt;twisted.roundup@...>)
  is not a member of any of the restrict_post groups.

If you do not wish the message to be posted, or have other concerns, 
please send a message to the list owners at the following address: 
  pgsql-jdbc-owner@...
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
spam :-(
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc commented|
|-|-|

In changeset f1e1e8aa870aff8c2dfb3dce9cb978a32624c998
```
#!CommitTicketReference repository="" revision="f1e1e8aa870aff8c2dfb3dce9cb978a32624c998"
Merge pull request #568 from twisted/8862-tox-extras-2

Author: derwolfe
Reviewer: rodrigc
Fixes: #8862

Use extras in tox.ini
```
